python/doc/examples/data_analysis/smoothing_mixture.ipynb

```diff
@@ -0,0 +1,524 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Sensitivity of bandwidth in kernel smoothing\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "We consider the distribution\n",
+    "\n",
+    "$$\n",
+    "f_1(x) = w_1 f_A(x) + w_2 f_B(x)\n",
+    "$$\n",
+    "\n",
+    "for any $x\\in\\mathbb{R}$ where $f_A$ is the density of the gaussian distribution \n",
+    "$\\mathcal{N}(0,1)$, $f_B$ is the density of the gaussian distribution \n",
+    "$\\mathcal{N}(3/2,(1/3)^2)$ and the weights are $w_1 = \\frac{3}{4}$ and $w_2 = \\frac{1}{4}$.\n",
+    "\n",
+    "This is a mixture of two gaussians: 1/4th of the observations have the $\\mathcal{N}(0,1)$ \n",
+    "distribution and 3/4th of the observations have the $\\mathcal{N}(3/2,(1/3)^2)$ distribution. \n",
+    "This example is considered in (Wand, Jones, 1994), page 14, Figure 2.3.\n",
+    "\n",
+    "We consider a sample generated from independent realizations of $f_1$ and \n",
+    "want to approximate the distribution from kernel smoothing. \n",
+    "More precisely, we want to observe the sensitivity of the resulting density on the \n",
+    "bandwidth.\n",
+    "\n",
+    "## References\n",
+    "\n",
+    "* \"Kernel Smoothing\", M.P.Wand, M.C.Jones. Chapman and Hall / CRC (1994).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simulate the mixture by merging two samples\n",
```

Comment:
```suggestion
    "## Generate the mixture by merging two samples\n",
```
@adutfoy Is that a situation where "generate" should be used instead of "simulate"?